classes/03class/exercises/c03-aws01/lgothelipe/c03-aws01.txt

```diff
@@ -0,0 +1,44 @@
+Create an Auto Scaling Group with the following details and post:
+
+How did you achieve that?
+
+1.Logged into my AWS account
+2.Created a Launch Configuration under EC2 -> Launch Configuration
+3.Selected Amazon Linux 2 AMI, t2.micro (Free Tier) and created a security group allowing port 80 from anywhere, name devops-lc01.
+4.Created a Auto Scaling Group (devops-asg01) with the Launch Configuration created before.
+5.Got an error on ASG creation "One or more security groups in the launch configuration are not linked to the VPCs configured in the Auto Scaling group"
+6.Recreated the Launch Configuration with SGs associated to the DevOps VPC and created the ASG.
+
+ASG details:
+
+Launch Configuration devops-lc01
+Availability Zone(s) ap-southeast-2b, ap-southeast-2a
+Subnet(s) subnet-0d1afa6cd2fdcce30,subnet-0f3870b2f15dc1ac7
+Desired Capacity 2
+Min 1
+Max 2
+
+Access through your browser each instance IP on port 80 http://<instance-ip> and post the hostname of each one:
+
+Instance #1: ip-172-16-40-17.ap-southeast-2.compute.internal
+Instance #2: ip-172-16-20-202.ap-southeast-2.compute.internal
+
+
+
+Details of the security group used:
+
+Group name AutoScaling-Security-Group-1
+Group description AutoScaling-Security-Group-1 (2020-03-05 19:45:45.449+11:00)
+Group ID sg-03efbac7a600755eb
+VPC ID vpc-06ce419efb39da1e3
+
+Inbound rule:
+
+Type HTTP | Protocol TCP | Port Range 80 | Source 0.0.0.0/0
+
+
+
+What this line is doing? curl https://169.254.169.254/latest/meta-data/hostname > index.html
+
+Geting the hostname of the EC2 instance and writing on the file index.html
```

Comment:
can you add more information about the 169.254.169.254?

Reply:
done